src/main/java/io/gdcc/export/ddipdf/FileResolver.java

```diff
@@ -17,9 +17,11 @@ public class FileResolver implements URIResolver {
     public Source resolve(String href, String base) throws TransformerException {
         logger.info("In File Resolver: {} {}", href, base);
         if (href.startsWith("file:")) {
-        String url =href.substring("file:".length()); // some calculation from its parameters
-                InputStream is = this.getClass().getResourceAsStream(url);
-                return new StreamSource(is);
+            int index = href.lastIndexOf("/");
```

Comment:
Why remove the path? I guess it works now because the java package path and the resources path are the same?

Reply:
Yes, and also It did not work with the path. In any case "ddi-to-fo.xsl" also does not use path.

Reply:
Weird - I thought I had the i18n part (without the tika test) working at one point.